resources/assets/js/helpers/dates.ts

```diff
@@ -0,0 +1,10 @@
+import dayjs from "dayjs";
+import LocalizedFormat from 'dayjs/plugin/localizedFormat';
+import "dayjs/locale/fr";
+import { Locales } from "./localize";
+
+dayjs.extend(LocalizedFormat)
+
+export const readableDateTime = (locale: Locales, date: Date): string => {
```

Comment:
How are we handling timezones with this implementation? Everything from the server would be UTC, correct?

Reply:
Good point, timezones are still ignored here.

Reply:
We should be able to [determine the timezone on the fly](https://stackoverflow.com/a/34602679) (if we want to display based on browser/system info), or we can display an explicit timezone like we're doing on the Twig side.

Reply:
Whoah! dayjs seems to be converting it from utc to my browsers local timezone, on its own!

Reply:
Could timezones actually be easy, for once??

Reply:
The question is, do we want that? In most other places we're displaying EST/EDT explicitly haha.

Reply:
> The question is, do we want that?
😭

Reply:
I'll try to get the right EST/etc code to display explicitly.

Reply:
Sounds like Gray thinks local time is fine.